.github/workflows/golangci-lint.yaml

```
@@ -29,4 +29,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@a4f60bb28d35aeee14e6880718e0c85ff1882e64 # v6.0.1
         with:
-          version: v1.58
+          version: v1.59.0
```

```suggestion
          version: v1.59
```

IIUC from @cpanato, we should drop the patch version, which lets us consume the latest patch version automatically